WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingDetailsViewController.swift

```diff
@@ -169,6 +176,20 @@ class NotificationSettingDetailsViewController: UITableViewController {
         return [section]
     }
 
+    private func sectionsForUnkownDeviceStream() -> [Section] {
+        defer {
+            WPAnalytics.track(.pushNotificationPrimerSeen, withProperties: [Analytics.locationKey: Analytics.alertKey])
```

Comment:
Is there a reason this is in a `defer`?

Reply:
I like to defer analytics calls _just in case_. I can remove it if you think it looks too weird 😉

Reply:
I don't really mind that much, I just wondered if it was really necessary :)

Reply:
Ok then I left them 😜